src/main/kotlin/org/ionproject/integration/model/external/timetable/OutputRepresentations.kt

```diff
@@ -9,7 +9,83 @@ data class TimetableDto(
     val programme: ProgrammeDto,
     val calendarTerm: String,
     val classes: List<ClassDto>
-)
+) {
+    companion object {
```

Comment:
I think we could reduce complexity a little here just so it's a little easier to read and make changes.
Though we shouldn't worry too much as we'll probably need to re-do all this later when we refactor the model objects.

Reply:
PR #192 merged